### PR TITLE
PreAuth: fixed success-redirect process on multi-website.

### PR DESCRIPTION
**Pull request by YevhenShBolt**
Also included code from PR https://github.com/BoltApp/bolt-magento2/pull/301

### CODE REVIEW

File: Block/Js.php [hunk 1]
```diff
@@ -443,16 +443,11 @@ public function shouldDisableBoltCheckout()
...
-            $quote = $this->checkoutSession->getQuote();
-        }
+        /** @var Quote $quote */
+        $quote = $this->checkoutSession->getQuote();
 
-        // TODO: need to check and add unit tests for case when we return null instead of zero!
-        // Several time i caught when quote was created but do not have store_id - it was null.
-        return (int) (($quote && $quote->getStoreId()) ?
-            $quote->getStoreId() : 0);
+        return  ($quote && $quote->getStoreId()) ?
```
Comment (matt-thomason) [inline Block/Js.php]: You removed the TODO here without creating the unit tests.

File: Controller/ReceivedUrl.php [hunk 2]
```diff
@@ -112,9 +112,10 @@ public function execute()
                 $logMessage = $noSuchEntityException->getMessage();
                 $this->logHelper->addInfoLog('NoSuchEntityException: ' . $logMessage);
 
-                $this->bugsnag->registerCallback(function ($report) use ($incrementId) {
+                $this->bugsnag->registerCallback(function ($report) use ($incrementId, $magentoStoreId) {
                     $report->setMetaData([
                         'order incrementId' => $incrementId,
+                        'MagentoStoreId'    => $magentoStoreId
```
Comment (matt-thomason) [inline Controller/ReceivedUrl.php]: Can we use `magento_store_id` as the metadata key to be consistent?
